app/src/modules/insights/dashboard-utils/apply-data-to-panels.ts

```diff
@@ -0,0 +1,51 @@
+import { Panel } from '@directus/shared/types';
+
+export const applyDataToPanels = (panels: Panel[], incomingData: Record<any, any>) => {
+	if (!incomingData) return panels;
+
+	const panelData: Record<string, Panel | Panel[]> = {};
+	const multiQueryPanelData: Record<string, any> = {};
```

Comment:
Lets make these names a tad more descriptive, or add some comments. Little confused what panelData is compared to "multi query" panel data 🤔